flightlib/include/flightlib/bridges/unity_message_types.hpp

```diff
@@ -54,6 +59,30 @@ struct Camera_t {
                            0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0};
 };
 
+struct EventCamera_t {
+  std::string ID;
+  // frame Metadata
+  int channels{3};
+  int width{346};
+  int height{260};
+  Scalar fov{70.0f};
+  Scalar depth_scale{0.20};  // 0.xx corresponds to xx cm resolution
+  // metadata
+  bool is_depth{false};
+  int output_index{0};
+  // eventcamera settings
+  float Cm{0.1};
+  float Cp{0.1};
```

Comment:
what are Cm and Cp? 
and why not using "Scalar" instead "float"

Reply:
cp and cm are the thresholds of the event sensor.